packages/frontend/src/features/sqlRunner/hooks/useVirtualView.tsx

```diff
@@ -29,20 +29,20 @@ const createCustomExplore = async ({
 /**
  * Create a virtual view (a.k.a. custom explore) - users can query from them in the Explore view
```

Comment:
kept the wording in comments bc I think it's useful and quick to understand what they actually are